modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/exception/DLReviewDateException.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.document.library.web.internal.exception;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+/**
+ * @author Brian Wing Shun Chan
+ */
+public class DLReviewDateException extends PortalException {
```

Comment:
Same thing as the previous exception (rename).

Reply:
The error is generic by default to obtain the date and all the exceptions have this kind of names
```
EntryExpirationDateException
ArticleExpirationDateException
QuestionExpirationDateException
```
But yes, the name is horrible...
Do you think that FileEntryExpirationDateException is a _little_ better?

Reply:
Sound good.